src/main/java/com/endava/endabank/exceptions/ControllerExceptionHandler.java

```diff
@@ -22,15 +31,67 @@ public ResponseEntity<ErrorMessage> resourceNotFoundException(ResourceNotFoundEx
         return new ResponseEntity<>(message, HttpStatus.NOT_FOUND);
     }
 
+    @ExceptionHandler(MethodArgumentNotValidException.class)
+    public ResponseEntity<Map<String, String>> handleValidationErrorException(MethodArgumentNotValidException ex, WebRequest request) {
+        Map<String, String> errors = new HashMap<>();
```

Comment:
Hi,
You can replace this block for this:

`Map<String, String> errors = ex.getBindingResult().getAllErrors().stream()
                .collect(Collectors.toMap(error -> ((FieldError) error).getField(), error -> error.getDefaultMessage()));`